packages/fx-core/templates/azure/config.bicep

```diff
@@ -0,0 +1,5 @@
+@secure()
```

Comment:
this seems to be a test file remove it as well as other test files?

Reply:
The test cases need to be updated to avoid writing files under root folder, as well as cleanup the test files after test case finished. You can talk with @wenytang-ms on how to update the test cases.

Reply:
It is fixed.